substrate-ui/src/App.js

```diff
@@ -50,39 +52,47 @@ function Main () {
   const contextRef = createRef();
 
   return (
-    <div ref={contextRef}>
-      <Sticky context={contextRef}>
-        <AccountSelector setAccountAddress={setAccountAddress} />
-      </Sticky>
-      <Container>
-        <Grid stackable columns='equal'>
-          <Grid.Row stretched>
-            <NodeInfo />
-            <Metadata />
-            <BlockNumber />
-            <BlockNumber finalized />
-          </Grid.Row>
-          <Grid.Row>
-            <GettersAndSetters accountPair={accountPair}/>
-          </Grid.Row>
-           <Grid.Row stretched>
-            <Balances />
-           </Grid.Row>
-           {/*<Grid.Row>*/}
-           {/* <Transfer accountPair={accountPair} />*/}
-           {/* <Upgrade accountPair={accountPair} />*/}
-           {/*</Grid.Row>*/}
-          <Grid.Row>
-            <Interactor accountPair={accountPair} />
-            <Events />
-          </Grid.Row>
-          <Grid.Row>
-            <TemplateModule accountPair={accountPair} />
-          </Grid.Row>
-        </Grid>
-      </Container>
-      <DeveloperConsole />
-    </div>
+   UI_Metrics_Chain
```

Comment:
Not sure where this line came from, but it's causing a syntax error. Could you remove it, please? Once this is removed, I think it's high time we merge this. It does introduce quite a few whitespace-related linter errors, but we can always fix those later - I don't want these features to fall by the wayside.